centos_ci.sh

```diff
@@ -292,14 +292,42 @@ function perform_release() {
   curl http://minibot.19cf262c.svc.dockerapp.io:9009/hubot/centosci -H "Content-Type: application/json" -d '{"payload":{"status":"success","message":'"\"$MESSAGE\""',"url":'"\"$URL\""'}}' || true
 }
 
+function perform_docs_publish() {
+  # REPO and BRANCH variables are populated via https://ci.centos.org/job/minishift-docs
+  REPO_OWNER=$(echo $REPO | cut -d"/" -f4)
+  cd gopath/src/github.com/minishift/minishift
+
+  # Get changes from remote repo/branch
+  if [[ "$REPO_OWNER" != "minishift" ]]; then
+    git remote add custom-remote $REPO
```

Comment:
I guess these git commands work, but they don't really make sense to me. Why adding a remote if this is a once-off operation. You add the remote as shortcut if you use it a lot of times.

And why do you fetch and then track? IMO the most straight forward way here is what you see also documented in pull requests when you click on the "command line instructions" link

    # create the branch to host the code you want to build
    git checkout -b docs_branch master 
    # pull the changes into this branch
    git pull $REPO $BRANCH

So why so complicated?

Reply:
@hferentschik , I tested that here https://ci.centos.org/job/minishift-pr/2803/console and found that during pull it somethings ask to merge the commit and wait for user's input.

Reply:
`git pull -f`?